test/test_p4est_2d.jl

```diff
@@ -529,7 +529,7 @@ end
                             0.11085870166618325,
                             0.23309905989870722,
                             0.13505351590735631,
-                            0.7932047512585592
+                            0.7932048824622121
```

Comment:
Do you know why you needed to adapt those tests? Is it because of the different order within computing the normal vectors?